nomenclature/core.py

```diff
@@ -15,37 +15,41 @@
 class DataStructureDefinition:
     """Definition of datastructure codelists for dimensions used in the IAMC format"""
 
-    def __init__(self, path):
+    def __init__(self, path, dimensions=["region", "variable"]):
         """
 
         Parameters
         ----------
         path : str or path-like
             The folder with the project definitions.
+        dimensions : list of str, optional
+            List of :meth:`CodeList` names, initialized from a sub-folder of `path`.
```

Comment:
```suggestion
            List of :meth:`CodeList` names, initialized from a sub-folder of `path` of the `CodeList` names.
```
or something along those lines to stress the fact that this attempts to read e.g. `path / "region"`

Reply:
Done